guides/assets/stylesheets/main.css

```diff
@@ -70,7 +70,7 @@ table {
 }
 
 table th, table td {
-  padding: 0.25em 1em;
```

Comment:
Overriden by [this line](https://github.com/rails/rails/pull/32401/files#diff-aa013ea6e60ea4f2d64b4b2583249bd4L700), so changed padding values.